src/Microsoft.Extensions.Configuration.AzureAppConfiguration/ConfigurationStoreConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal class ConfigurationStoreConstants
+    {
+        public static readonly string ConfigStoreNameReplicaSeparator = "---";
```

Comment:
> static readonly [](http://example.com/codeflow?start=14&length=16)

We can make it `const` instead of `static readonly`

Reply:
With the new implementation, it is not needed anymore.